README.rst

```diff
@@ -79,24 +79,24 @@ Clone the repository and install it in editable mode with all the extras:
 
 .. code-block:: bash
 
-   git clone git@github.com:iqm-finland/cirq-on-iqm.git
-   cd cirq-on-iqm
-   pip install -e ".[dev,docs]"
+   $ git clone git@github.com:iqm-finland/cirq-on-iqm.git
+   $ cd cirq-on-iqm
+   $ pip install -e ".[dev,docs,testing]"
 
 
 Build and view the docs:
 
 .. code-block:: bash
 
-   python setup.py docs
-   firefox build/sphinx/html/index.html
+   $ tox -e docs
```

Comment:
Following these steps I get the following error

> Warning, treated as error:
> html_static_path entry '_static' does not exist
> ERROR: InvocationError for command 'cirq-on-iqm\.tox\docs\Scripts\sphinx-build.EXE' -b html docs 'cirq-on-iqm/build/sphinx/html' -W (exited with code 2)
> ________________________ summary ____________________________
> ERROR:   docs: commands failed

Reply:
Fixed.